xicam/core/execution/workflow.py

```diff
@@ -599,6 +599,41 @@ def __init__(self, name="", operations=None):
 
         self.lastresult = []
 
+    def clone(self):
```

Comment:
Did you happen to write a test for this?

Reply:
Not yet, the new clone operation in this code does not work, I had to revert back to my old code. I will talk to Ron about a better solution and push a new commit.

Reply:
Rewritten; also added tests